libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -21,7 +21,7 @@
         {{ b.title }}
       </div>
       <div class="book--content">
-        <div class="book--content--cover">
+        <div class="book--content--cover" role="img" aria-label="cover page">
```

Comment:
aria-label attribute is used to define a string that labels the current element.